capstone-frontend/public/index.html

```diff
@@ -24,7 +16,12 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <title> BookBook </title> 
+    <link rel="shortcut icon" href="https://pbs.twimg.com/profile_images/813056632250925057/t-DDGecT_400x400.jpg"/>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossOrigin="anonymous"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```

Comment:
I probably not be using Jquery but I guess is nice to have. 👍

Reply:
We need Jquery as a dependency for bootstrap, which is why I included it here. But if we move away from Bootstrap I'll remove.